tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -155,8 +155,8 @@
 
     <!-- Integration: Microsoft.Azure.Cosmos.Client -->
     <!--    Assembly: Microsoft.Azure.Cosmos.Client -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.43.1 -->
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.43.1" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.44.0 -->
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.44.0" />
```

Comment:
Also untested

Reply:
I'll toss it on the backlog to test

I'll also remove this one from being bumped automatically until it is actually tested on CI automatically